jwst/white_light/white_light.py

```diff
@@ -40,9 +40,20 @@ def white_light(input):
     log.debug("norders = %d, sporders = %s, ntables_order = %s",
               norders, str(sporders), str(ntables_order))
 
-    # Compute the flux sum for each integration in the input
+    # Create a wavelength mask, using cutoffs if specified, then
+    # compute the flux sum for each integration in the input.
+    low_cutoff = -1.
+    high_cutoff = 1.e10
+    if min_wave is not None:
+        low_cutoff = min_wave
+    if max_wave is not None:
+        high_cutoff = max_wave
+
     for i in range(ntables):
-        fluxsums.append(np.nansum(input.spec[i].spec_table['FLUX']))
+        wavelength_mask = np.where((input.spec[i].spec_table['WAVELENGTH']>low_cutoff)&
```

Comment:
Science question, which I don't know the answer, maybe @hbushouse: Should the cutoffs be inclusive or exclusive? IMHO don't really care, as long as documentation is clear.

Reply:
As a user I would interpret them as being the min and max wavelength values to *include* in the summation, hence inclusive.

Reply:
I agree with this, yet didn't write it that way. Oops!